microsoftgraph/msgraph-cli-archived
# Task/docker container

## Overview

1. Adds documentation on running mg inside a docker container
2. Removes `build_extensions.py` because it is now redundant
 
### Demo

N/A

### Notes

Users won't be able to sign in when using mg inside a docker container. See #21 

## Testing Instructions

* Install Docker
* Pull these changes
* Run `docker build --target build-env -f ./build_scripts/debian/Dockerfile -t microsoft/msgraph-cli:ubuntu-builder .` from project root
* Run `docker run -it --name deb microsoft/msgraph-cli:ubuntu-builder` to start the container in interactive mode.
* Type `mg` and observe that it works


Fixes #35, #31
